app/assets/javascripts/angular/services/repo_service.js

```diff
@@ -0,0 +1,17 @@
+app.service('Repo', ['$resource', function($resource) {
+    this.getRepoData = function(owner, repo){
+    var repo = $resource('/api/v1/repos', {owner: "@owner", repo: "@repo"});
+    .success(function (data){
```

Comment:
Expected an identifier and instead saw '.'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'success' is not defined.<br>'data' is defined but never used.